connectors/http/graphql/element-templates/graphql-outbound-connector-hybrid.json

```diff
@@ -0,0 +1,469 @@
+{
+  "$schema" : "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name" : "GraphQL Outbound Connector",
+  "id" : "io.camunda.connectors.GraphQL.v1",
+  "description" : "Execute GraphQL query",
+  "documentationRef" : "https://docs.camunda.io/docs/components/connectors/protocol/graphql/",
+  "version" : 5,
```

Comment:
Does hybrid need to be version 5?

Reply:
Its coming from the generator. I dont think we support custom version per generated output cc @chillleader